### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by lwd01** [shuaichaochao/HybridHash#4]
Hello author, when I reproduced the CIFAR dataset, I found that the loss kept hovering at 0.54, and then the test found that the map value was about 0.45, so I would like to ask the details of the training here, and is the pre-training weight from my own training?

**Comment by shuaichaochao** [comment]
The training parameters are the same as in the train.py file, and the download link for the pre-training weights is on the fourteenth line in the HybridHash.py file, please make sure to load it.
